Library/docs/source/learn/physical_digital_realities/fiducial_markers.md

```diff
@@ -0,0 +1,249 @@
+
+```
+// Assuming an asset collection and asset identifiers for a fiducial marker image...
+csp::systems::AssetCollection MarkerAssetCollection = ...;
+csp::systems::Asset MarkerAsset = ...;
+
+// Associate the marker component with the asset.
+FiducialMarkerComponent->SetAssetCollectionId(MarkerAssetCollection.Id);
+FiducialMarkerComponent->SetMarkerAssetId(MarkerAsset.Id);
+```
+
+### Setting Transform Properties
+
+The transform defines the fiducial marker's position, orientation, and scale in the digital coordinate system. Ensure the scale matches the marker's real-world dimensions.
+
+```
+// Set the transform to define position, rotation, and scale.
+SpaceTransform MarkerComponentTransform =
+{
+    csp::common::Vector3(1.0, 2.0, 3.0),
+    csp::common::Vector4(0.0, 0.0, 0.0, 1.0),
+    csp::common::Vector3(0.5, 0.5, 0.01)
+};
+
+FiducialMarkerComponent->SetTransform(MarkerComponentTransform);
+```
+
+### Updating the Entity
+
+Queue the entity for a network update to replicate the marker's configuration across connected devices.
+
+```
+Entity->QueueUpdate();
+```
+
+### Example
+
+Here is a complete example of creating and configuring a fiducial marker component:
+
+```
+// Create an entity to hold the fiducial marker.
+SpaceTransform MarkerEntityTransform =
+{
+    csp::common::Vector3::Zero(),
+    csp::common::Vector4::Zero(),
+    csp::common::Vector3::One()
+};
+
+csp::multiplayer::SpaceEntitySystem::EntityCreatedCallback Callback = [](csp::multiplayer::SpaceEntity* CreatedSpaceEntity)
+{
+    if (CreatedSpaceEntity)
+    {
+        // Add fiducial marker component.
+        FiducialMarkerSpaceComponent* MarkerComponent =
+            static_cast<FiducialMarkerSpaceComponent*>(CreatedSpaceEntity->AddComponent(ComponentType::FiducialMarker));
+        
+        // Set marker properties.
+        MarkerComponent->SetAssetCollectionId("AssetCollection123");
+        MarkerComponent->SetMarkerAssetId("MarkerAsset456");
+
+        SpaceTransform Transform = 
+        {
+            csp::common::Vector3(1.0, 2.0, 3.0),
+            csp::common::Vector4(0.0, 0.0, 0.0, 1.0),
+            csp::common::Vector3(0.5, 0.5, 0.01)
+        };
+        MarkerComponent->SetTransform(Transform);
+        
+        // Queue the entity for network update.
+        CreatedSpaceEntity->QueueUpdate();
+    }
+};
+
+EntitySystem->CreateObject("MyFiducialMarker", MarkerEntityTransform, Callback);
+```
+
+## Relating Fiducial Marker Transforms to AR Spaces
+
+Fiducial markers are critical in aligning digital content with physical spaces in augmented reality (AR). Their transforms-defined by position, rotation, and scale-determine the placement and orientation of AR content. Understanding how to manage fiducial marker transforms ensures precise spatial alignment in CSP applications.
```

Comment:
[nit] "transforms-defined by position, rotation, and scale-determine": this should be done with an em-dash, not a normal dash.

Reply:
Elected to strip out the pos,rot,scale part as the term 'transform' implicitly covers it.

Reply:
Resolved in latest.